pkgs/applications/version-management/gogs/default.nix.patch

```diff
@@ -0,0 +1,22 @@
+2,3c2
```

Comment:
I immediately don't see any better way than this.

I also didn't know about `go2nix`, so perhaps its users might have a better idea how to handle such a situation. Maybe @cstrahan?